### DISCUSSION TITLE
SEO only via SSR

### DISCUSSION THREAD

**Discussion by BlackYuzia** [nuxt/nuxt#29220] [Questions]
Greetings.
Could we somehow load SEO meta tags via SSR mode and then use CSR?

I don't mean:
- User do request
- User get full HTML via SSR
- User do hydration to sync with HTML

I mean:
- User do request
- User get partial HTML only with SEO (meta tags)
- User get next part HTML like JS, CSS etc
- User run JS (without hydration, cuz we don't get in HTML except meta tags as well)

**Comment by BlackYuzia** [comment]
Suppose, `<OnlyClient>` that I was want to... but not sure :eyes:

**Comment by Samrexs** [comment]
It’s encouraging to see everyone discussing their experiences with Nuxt 3! I recently read some Pearl Lemon reviews highlighting the importance of clear documentation and user-friendly setups. I think providing more examples for TypeScript configurations could really enhance the onboarding process for new developers. Looking forward to any updates from the team
